CMakeLists.txt

```diff
@@ -396,7 +387,7 @@ if (NOT CLANG_VERSION_LESS_15)
 endif()
 
 # For use by hipcc
-execute_process(COMMAND ${LLVM_CONFIG} --host-target
+execute_process(COMMAND ${LLVM_CONFIG_BIN} --host-target
```

Comment:
Yeah I wanted to make this change since sometimes it's not clear when the project wants you to specify the full path to the binary vs when just the directory is needed.